src/asciidoc/docs/guidelines/persistSessionData.adoc

```diff
@@ -0,0 +1,64 @@
+= Persisting data across sessions
+
+:page-slug: persist-session-data
+:page-description: How to persist data that doesn't expire with the session.
+
+Some templates need to store data about the user's session, such as a user's name or game score.
+You can store data temporarily using `window.sessionStorage`.
+However, data stored in this way expires when the session ends.
+If you need to persist data across sessions, `window.localStorage` can be used.
```

Comment:
Still passive. You could just say "... use `window.localStorage`."

Reply:
Done